pagespeed/envoy/envoy_cluster_manager.h

```diff
@@ -0,0 +1,68 @@
+
+#ifndef PAGESPEED_ENVOY_CLUSTER_MANAGER_H_
```

Comment:
nit: just stick with #pragma

Reply:
Removed #ifndef and kept #pragma only.
Code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/05e80e98faba31ff5a94f1244b1833fa88a1fae4